src/main/java/seedu/address/model/house/HousingType.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.model.house;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+
+/**
+ * Represents a House's housing type.
+ * Guarantees: immutable; is valid as declared in {@link #isValidLevel(String)}
+ */
+public class HousingType {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "HousingType should only be Landed or NonLanded";
```

Comment:
I think the message here should be something along the line of `should only be HDB, Condominium, Landed`.

Reply:
yup, agreed